scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -382,12 +382,22 @@
             :to-make-room-for pending-job)]
     [state nil]))
 
+(defn reserve-hosts! [state preemption-decisions]
+  (let [multiple-tasks (filter #(< 1 (count (:task %))) preemption-decisions)
+        reservations (into {} (map (fn [d] [(:job/uuid (:to-make-room-for d))
+                                            (:hostname d)])
+                                   multiple-tasks))]
```

Comment:
Nitpick: This  block would be easier to read for me if we used `->>` of if we had a `(map-from-seq fk fv es)` in our utils.

Reply:
Done